tests/lib/tools/str-tool

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+show_help() {
+    echo "usage: str-tool to-one-line [string]"
```

Comment:
my preference would be to have just a "to-one-line" tool directly

Reply:
Sure, thanks for the suggestion